packages/commonwealth/server/util/roles.ts

```diff
@@ -237,6 +262,42 @@ export async function createRole(
   return new RoleInstanceWithPermission(
     role_assignment.toJSON(),
     chain_id,
-    role_name
+    role_name,
+    community_role.allow,
+    community_role.deny
   );
 }
+
+// Permissions Helpers for Roles
+/// ////////////////////////////////////////////////////////////////////////////////////////////
+
+export enum PermissionError {
+  NOT_PERMITTED = 'Action not permitted',
+}
+
+// eslint-disable-next-line no-bitwise
+export const BASE_PERMISSIONS: Permissions = BigInt(1 << Action.CREATE_THREAD);
```

Comment:
This will also not work as in my above comment -- see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt for better ways to declare this.